biz.aQute.bndlib/src/aQute/bnd/osgi/resource/CapReq.java

```diff
@@ -20,26 +22,27 @@
 	private final Resource				resource;
 	private final Map<String,String>	directives;
 	private final Map<String,Object>	attributes;
+	private transient int				hashCode	= 0;
```

Comment:
Well it can be, but the value transient and can be recomputed from the other data of the object. It is essentially a cached value.